lib/ravelin/authentication_mechanisms/magic_link.rb

```diff
@@ -0,0 +1,27 @@
+module Ravelin
+  module AuthenticationMechanisms
+    class MagicLink < RavelinObject
+      DELIVERING_METHODS = %w(email sms)
```

Comment:
Please change this to `TRANSPORTATION_MECHANISM` (how Ravelin refers to this)

Reply:
Fixed, thanks